ccip-cli/src/commands/manual-exec.ts

```diff
@@ -88,6 +88,12 @@ export const builder = (yargs: Argv) =>
         describe:
           'Clears buffers (if a previous attempt was aborted) or any ALT owned by this sender.',
       },
+      'receiver-object-ids': {
+        type: 'array',
+        describe: 'Receiver object IDs for Sui execution (if executing on Sui destination)',
+        string: true,
+        example: '--receiver-object-ids 0xabc... 0xdef...',
+      },
```

Comment:
Is this really needed? I see `SuiExtraArgsV1` has a `receiverObjectIds` property;

Isn't then the user expected to pass those as extraArgs at request time? If so, it'll be available inside the `request`, and you should not need to receive them explicitly here.

Anything else needed but not passed, should be auto-derivable.

Is there any valid usecase when payer would need to pass additional receiverObjectIds beyond those providev by user or hardcodable/auto-derivable?

CLI strives to provide minimal options, and to do the extra mile to self-drive/discover anything that may be needed, instead of providing users with yet another knob they'll need to read through to understand how to use the tool.

Reply:
I'm not a big fan of overrides like this, but I think we can let it pass while this is in development; we should clean it up though once the contracts and algorithms stabilize and are shown to be working